Website/DesktopModules/Hotcakes/Core/Controllers/ProductsController.cs

```diff
@@ -834,6 +834,15 @@ private LineItem ConvertProductToLineItem(ProductPageViewModel model)
 
             var li = model.LocalProduct.ConvertToLineItem(HccApp, model.Quantity, model.Selections, userPrice);
 
+            if (model.LocalProduct.AllowUpcharge)
+            {
+                var boolCoverCreditCardFees = Request.Form["covercreditcard"] == "on";
```

Comment:
@ArielBlanco1990 Please give this control another selector name, such as `upchargeallowed` or something.  Leave the term "credit card" out of it.  That is only one possible use case for this feature.  😊

Reply:
@ArielBlanco1990 This should be done in the viewsets as well.

Reply:
@WillStrohl This is now resolved.